Anagram.java

```diff
@@ -28,22 +30,53 @@ public static void main(String args[]) {
 
 	// Returns true if the two given strings are anagrams, false otherwise.
 	public static boolean isAnagram(String str1, String str2) {
-		// Replace the following statement with your code
-		return false;
+	str1 = str1.replaceAll("[^a-zA-Z]", "").toLowerCase();
+	str2 = str2.replaceAll("[^a-zA-Z]", "").toLowerCase();
+	
+	if (str1.length() != str2.length()) {
+    return false;
+	}
+		char[] arr1 = str1.toCharArray();
+		char[] arr2 = str2.toCharArray();
+	
+		Arrays.sort(arr1);
```

Comment:
Both Arrays.sort() and string.toCharArray() weren't taught in recitation or lecture, therefore each of those is hw guidelines violation. -4